pmlc/dialect/stripe/into_mlir.cc

```diff
@@ -212,6 +212,7 @@ static void IntrinsicIntoMLIR(OpBuilder* builder, SymbolTable* locals, const str
     throw std::runtime_error("Unknown intrinsic: " + intrinsic.name);
   }
   auto genericBuilder = abstractOp->getInterface<util::GenericBuilder>();
+  IVLOG(1, "genericBuilder");
```

Comment:
It's OK to add IVLOG messages, but level 1 is usually reserved for messages that might of of interest to most users, and debug level logs should be lowered to 3 or 4 for commit.